libs/content/workspace-ffi/src/apple/ios/api.rs

```diff
@@ -275,6 +285,12 @@ pub unsafe extern "C" fn end_of_document(obj: *mut c_void) -> CTextPosition {
 #[no_mangle]
 pub unsafe extern "C" fn touches_began(obj: *mut c_void, id: u64, x: f32, y: f32, force: f32) {
     let obj = &mut *(obj as *mut WgpuWorkspace);
+
+    let frame = obj.context.frame_nr();
+    let pos = egui::pos2(x, y);
+    let span = span!(target: "svg_editor", Level::TRACE, "UIEvent touch start", ?pos, frame);
```

Comment:
I think most if not all of your tracing invocations can get cleaned up by using the instrument macro like we do in LB: https://docs.rs/tracing/latest/tracing/attr.instrument.html

Includes printing of arguments automatically and other such a nice features

Reply:
oh nice didn't know about that. I implemented it, thanks!